src/components/MessagePlainTextBody.vue

```diff
@@ -1,6 +1,7 @@
 <template>
-	<div>
-		<div id="mail-content" v-html="nl2br(enhancedBody)" />
+	<div id="mail-content">
+		<MdnRequest :message="this.message" />
```

Comment:
```suggestion
		<MdnRequest :message="message" />
```

no `this` in templates :)